src/Indicator.vala

```diff
@@ -125,11 +126,24 @@ public class DateTime.Indicator : Wingpanel.Indicator {
             menuitem_label.wrap_mode = Pango.WrapMode.WORD_CHAR;
             menuitem_label.xalign = 0;
 
-            var menuitem_box = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 6);
+            var menuitem_times = new Gtk.Label ("");
+            menuitem_times.set_markup (e.get_event_times ());
+            menuitem_times.hexpand = true;
+            menuitem_times.lines = 3;
```

Comment:
We shouldn't need any of this for times, they'll always fit on one line since they'll be a maximum of like 20 characters